src/OrleansRuntime/Streams/QueueBalancer/ConsistentRingQueueBalancer.cs

```diff
@@ -6,20 +6,26 @@
 
 namespace Orleans.Streams
 {
-    internal class ConsistentRingQueueBalancer : IAsyncRingRangeListener, IStreamQueueBalancer
+    public class ConsistentRingQueueBalancer : IAsyncRingRangeListener, IStreamQueueBalancer
```

Comment:
y make this, and all that it requires, public.  This is can/should remain internal, right?